### PR TITLE
adding missing dataField Condition for studyList page.

### PR DESCRIPTION
**Pull request by iksheth**

### CODE REVIEW

**Review by jonkiky — APPROVED** [review]
LGTM!

File: src/utils/tables.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ export function getColumns(tableConfig, classes, data, externalLinkIcon, linkto,
                   {' '}
                   {column.dataFromRoot ? data[column.dataField]
                     : (column.formatBytes ? formatBytes(value)
-                      : column.dataField === 'num_subjects'
+                      : (column.dataField === 'num_subjects' || column.dataField === 'numberOfCases')
```
Comment (jonkiky) [inline src/utils/tables.js:42]: The implementation fixes the bug correctly.  But will be possible to change the hardcode? If that takes long would not be fix in this release, can we have a ticket in the parking lot to keep the track of this?

Comment (iksheth) [reply src/utils/tables.js:42]: No we dont have any ticket. and if we change it in backend API we wont need this change. 
Screen Shot 2020-12-03 at 1 51 47 PM

**Comment by AjayDoddapaneni** [comment]
@iksheth @jonkiky Is it ready to merge?

**Comment by iksheth** [comment]
Yes 👍🏻

**Comment by iksheth** [comment]
Yes @AjayDoddapaneni its ready to merge

**Pull request merged**